options/locale/locale_en-US.ini

```diff
@@ -1512,7 +1512,9 @@ issues.label_description = Description
 issues.label_color = Color
 issues.label_exclusive = Exclusive
 issues.label_archive = Archive Label
-issues.label_archive_tooltip= Archived labels are excluded from the label search when applying labels to an issue. Existing labels on issues remain unaffected, allowing you to retire obsolete labels without losing information.
+issues.label_archived  = Archived
```

Comment:
Don't we already have this translation key?
Can we perhaps move the already existing `<no section>.archived` into this translation?